app/controllers/assignments_controller.rb

```diff
@@ -658,7 +669,49 @@ def upload_config_files
 
   private
 
-  # Build the tag/criteria file specified by +hash_to_build+ found in +zip_file+
+  # Configures the starter files for an +assignment+ provided in the +zip_file+
+  def config_starter_files(assignment, zip_file)
+    assignment.is_peer_review? ? starter_config_file = :peer_review_starter_files : starter_config_file = :starter_files
+    starter_file_settings = build_hash_from_zip(zip_file, starter_config_file).symbolize_keys
+    assignment.starter_file_type = starter_file_settings[:starter_file_type]
+    assignment.starter_files_after_due = starter_file_settings[:allow_starter_files_after_due]
```

Comment:
Just to clarify, you're *not* setting this as part of the assignment properties and you *are* assigning this here or you're doing it in both places?

Reply:
Yes that is correct. Stater file settings will be set from information in this file and will no longer be provided in the properties file